examples/web/index.html

```diff
@@ -245,6 +250,65 @@ <h3>Stacked blocks</h3>
           </ul>
         </div>
       </section>
+
+      <section>
+        <div class="Container">
+          <h2>Helpers</h2>
+          <h3>Typography</h3>
+          <h4 class="typography-heading-xlarge-text">Heading XLarge Text</h4>
+          <a href="#" class="typography-heading-xlarge-link">Heading XLarge Link</a>
+          <h4 class="typography-heading-large-text">Heading Large Text</h4>
+          <a href="#" class="typography-heading-large-link">Heading Large Link</a>
+          <h4 class="typography-heading-medium-text">Heading Medium Text</h4>
+          <a href="#" class="typography-heading-medium-link">Heading Medium Link</a>
+          <h4 class="typography-heading-small-text">Heading Small Text</h4>
+          <a href="#" class="typography-heading-small-link">Heading Small Link</a>
+          <h4 class="typography-heading-xsmall-text">Heading XSmall Text</h4>
+          <a href="#" class="typography-heading-xsmall-link">Heading XSmall Link</a>
+          <p class="typography-body-large-text-regular">
+            Body Large Text Regular
+          </p>
+          <p class="typography-body-large-text-bold">
+            Body Large Text Bold
+          </p>
+          <p class="typography-body-large-text-italic">
+            Body Large Text Italic
+          </p>
+          <p>
+            <a href="#" class="typography-body-large-text-link">Body Large Text Link</a>
+          </p>
+          <p class="typography-body-large-button">
+            Body Large Button
+          </p>
+          <p class="typography-body-medium-text-regular">
+            Body Medium Text Regular
+          </p>
+          <p class="typography-body-medium-text-bold">
+            Body Medium Text Bold
+          </p>
+          <p class="typography-body-medium-text-italic">
+            Body Medium Text Italic
+          </p>
+          <p>
+            <a href="#" class="typography-body-medium-text-link">Body Medium Text Link</a>
+          </p>
+          <p class="typography-body-medium-button">
+            Body Medium Button
+          </p>
+          <p class="typography-body-small-text-regular">
+            Body Small Text Regular
+          </p>
+          <p class="typography-body-small-text-bold">
```

Comment:
This one is obviously not bold, and the italic is missing:

<img width="216" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/156778758-9cca8cc5-59da-4be6-be63-3919f429c971.png">

It seems to be wrong in Figma:

<img width="492" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/156778625-63fce637-357b-4694-bebe-4c81787effad.png">